test/module/irohad/ametsuchi/kv_storage_test.cpp

```diff
@@ -105,7 +105,7 @@ class KVTest : public AmetsuchiTest {
 
   std::string domain_id = "ru";
   std::string account_name1 = "user1";
-  std::string account_data1 = R"({"age": "30"})";
+  std::string account_data1 = R"({"user1": {"age": "30"}})";
```

Comment:
`model::CreateAccount` should not contain JSON data field by requirements.

Reply:
Done